src/go/docker-credential-none/dcnone/dcnone_test.go

```diff
@@ -0,0 +1,93 @@
+package dcnone
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+func TestDCNoneHelper(t *testing.T) {
+	helper := DCNone{}
+
+	const server1 = "https://foobar.docker.io:2376/v1"
+	const server2 = "https://foobar.docker.io:9999/v2"
+	sawServers := map[string]bool{
+		server1: false,
+		server2: false,
+	}
+	creds := &credentials.Credentials{
+		ServerURL: server1,
+		Username:  "nothing",
+		Secret:    "isthebestmeshuggahalbum",
+	}
+
+	helper.Add(creds)
+
+	creds.ServerURL = server2
+	helper.Add(creds)
+
+	credsList, err := helper.List()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for server, username := range credsList {
+		_, ok := sawServers[server]
```

Comment:
Sorry to put you through another round of review, but when I ran the tests (by being in src/go/docker-credential-none/ and running `go test ./dcnone`) I got the following failure:
```
rancher-desktop\src\go\docker-credential-none> go test ./dcnone
--- FAIL: TestDCNoneHelper (0.00s)
    dcnone_test.go:79: Failed to store server https://foobar.docker.io:2376/v1
FAIL
FAIL    github.com/rancher-sandbox/rancher-desktop/src/go/docker-credential-none/dcnone 0.196s
FAIL
```
I'm running on Windows if it's relevant. I traced the problem to the call to `helper.List()` returning and empty `map[string]string`. This is because `~/.docker/plaintext-credentials.config.json` isn't there. That's as far as I got.